.github/workflows/test.yml

```diff
@@ -110,6 +180,18 @@ jobs:
       run: |
         . ci/ciLibrary.source
         ccda_build
+    - name: Save node cache
+      if: ${{ matrix.configurations.save_node_cache == 'true' }}
+      uses: actions/cache/save@v4
+      with:
+        path: ${{ steps.npm-cache-dir.outputs.dir }}
+        key: ${{ runner.os }}-node-${{ matrix.configurations.node_version }}-${{ hashFiles('**/package-lock.json') }}
+    - name: Save composer cache
+      if: ${{ matrix.configurations.save_composer_cache == 'true' }}
```

Comment:
@adunsulag I was getting the same error in test.yml as you saw in the inferno tests, and I think it was a collision when too many jobs simultaneously tried to save the same cache/key. If you still can't get it to work on a retry, maybe we should just try to restore the cache, but never try to save it, for inferno.